### PR TITLE
fix: remove padding when calc bbox of Text

### PR DESCRIPTION
**Pull request by xiaoiver**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / Document optimization
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Performance improvement
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

#753 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: packages/g-base/src/bbox/text.ts [hunk 1]
```diff
@@ -43,12 +43,11 @@ export default function (shape: IShape): SimpleBBox {
...
-    const padding = fontSize * 0.2;
     bbox = {
-      x: point.x - padding,
-      y: point.y - padding,
-      width: width + padding,
-      height: height + padding,
+      x: point.x,
+      y: point.y,
+      width: width,
+      height: height,
```
Comment (hustcc) [inline packages/g-base/src/bbox/text.ts:50]: 这个回滚回去的话，g6 那边的问题还是没有办法解决吧。

Comment (visiky) [reply packages/g-base/src/bbox/text.ts:50]: 增加一个属性为 padding 是否可以，如下
```ts
addShape('text', {
     text: 'xx',
     padding: [0, 0, 0, 0]
});
```

**Comment by visiky** [comment]
image

File: packages/g-base/src/bbox/text.ts (hunk 1 above)
Comment (hustcc) [reply packages/g-base/src/bbox/text.ts:50]: @Yanyan-Wang 十吾帮忙看看这个如果改回来，你们那边是不是会出问题？

**Review by visiky — APPROVED** [review]

**Pull request merged**